cypress/fixtures/HESS_test_event_test_promising.txt

```diff
@@ -0,0 +1,69 @@
+<VOEvent xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.ivoa.net/xml/VOEvent/v2.0 http://www.ivoa.net/xml/VOEvent/VOEvent-v2.0.xsd" version="2.0" role="test" ivorn="ivo://HESS/GRB#GRB_ToO220413">
+    <Who>
+        <Description>VOEvent created with voevent-parse, version 1.0.1. See https://github.com/timstaley/voevent-parse for details.</Description>
+        <AuthorIVORN>ivo://HESS</AuthorIVORN>
+        <Date>2022-04-13T12:47:02</Date>
+        <Author>
+            <title>HESS GRB Alert</title>
+            <contactName>HESS Collaboration</contactName>
+        </Author>
+    </Who>
+    <What>
+        <Param name="zenith_angle_min" unit="deg" value="52.097359981281" />
+        <Param name="zenith_angle_mean" unit="deg" value="55.536884423208" />
+        <Param name="zenith_angle_max" unit="deg" value="58.976408865135" />
+        <Param name="observation_duration" unit="s" value="2803.0" />
+        <Param name="significance" value="0.9674438609490041" />
+    </What>
+    <WhereWhen>
+        <ObsDataLocation>
+            <ObservatoryLocation id="GEOLUN" />
+            <ObservationLocation>
+                <AstroCoordSystem id="UTC-FK5-GEO" />
+                <AstroCoords coord_system_id="UTC-FK5-GEO">
+                    <Time unit="s">
+                        <TimeInstant>
+                            <ISOTime>2022-04-12T17:06:48.800000</ISOTime>
+                        </TimeInstant>
+                    </Time>
+                    <Position2D unit="deg">
+                        <Name1>RA</Name1>
+                        <Name2>Dec</Name2>
+                        <Value2>
+                            <C1>320.7943</C1>
+                            <C2>-0.2232</C2>
```

Comment:
Change this to be -65 or lower in declination so that the test will always pass.